pantab/_writer.py

```diff
@@ -3,12 +3,12 @@
 import shutil
 import tempfile
 import uuid
-from typing import Dict, List, Sequence, Tuple, Union
+from typing import Callable, Dict, List, Optional, Sequence, Tuple, Union
 
+import libwriter  # type: ignore
```

Comment:
Yep. Just FYI this was picked up and reordered when I ran isort **/*.py. Do you think it's worth raising an issue to look at adding ``# isort:skip`` or something to prevent this going forward?